### ISSUE TITLE
If device's FriendlyName contains single quote, Map is not displayed

### ISSUE THREAD

**Issue by nbogojevic** [yllibed/Zigbee2MqttAssistant#62]
There is a syntax error in Map.js caused by extraneous single quote in this case and that breaks display of the map. 

I'd say bug comes from line  https://github.com/yllibed/Zigbee2MqttAssistant/blob/e68ff9322095b04b9c017ad86c7310027e332c97/Zigbee2MqttAssistant/Views/Home/Map.cshtml#L73

Probably should be something like this (my C# is little bit rusty)

```C#
                WriteLiteral("{id: '" + device.ZigbeeId + "', label: '" + device.FriendlyName.Replace("'", @"\'") + "', shape:'" + getShape(device) + "', mass:" + getMass(device) + "},\n");
``` 

Thanks for the good work!

**Comment by carldebilly** [comment]
@all-contributors please add nbogojevic to bugs

**Comment by carldebilly** [comment]
@all-contributors please add nbogojevic to code

**Comment by carldebilly** [comment]
Fixed in Zigbee2MqttAssistant v0.2.0-dev.131
